src/classwork/04_assign/main.cpp

```diff
@@ -10,6 +12,27 @@ factorial.  Also, loop continues as long as user wants to.
 */
 int main() 
 {
+	char choice;
+	int num = 0;
+
+    do
+    {
+       cout<<"Please enter a number";
+       cin>>num;
+	   while(num > 0 && num < 11)
+	   {
+		   cout<<"Please enter another number";
+		   cin>>num;
+	   }
+
+	   factorial(num);
+	   cout<<"Type the letter 'Y' to continue";
+
+    } 
+    while (!(choice == 'y' || choice == 'Y'));
```

Comment:
**Code should be as follows:**

 do
    {
       cout<<"Please enter a number";
       cin>>num;
	   while(num < 1 || num > 10)
	   {
		   cout<<"Please enter another number";
		   cin>>num;
	   }

	   cout<<factorial(num)<<"\n";
	   cout<<"Type the letter 'Y' to continue";
	   cin>>choice;

    } 
    while (choice == 'y' || choice == 'Y');